evals/metrics/utils.py

```diff
@@ -84,10 +85,13 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+tokenizer = lambda text: list(jieba.cut(text))
```

Comment:
Do we need to specify a tokenizer here?

Reply:
It's a common function for bleu and rouge metrics.